Recipes/Recipes.DataAccess/Repositories/RecipesRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Recipes.DataAccess.Entities;
+
+namespace Recipes.DataAccess.Repositories
+{
+    public class RecipesRepository : GenericRepository<RecipeEntity>, IGenericRepository<RecipeEntity>
```

Comment:
`GenericRepository` already implements the `IGenericRepository`. There is no need to extend the RecipesRepository with it here brother

Reply:
the recipes repository overrides some methods. Most notably, it includes tags when querying recipes.

Reply:
Yes, you can still override them without the `IGenericRepository` interface